drivers/include/mlx90109.h

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (C) 2017 Jonas Radtke <jonas.radtke@haw-hamburg.de> <jonas@radtke.dk>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+ 
+ /**
+ * @defgroup    drivers_MLX90109 Read RFID Animal Tags
+ * @ingroup     drivers
+ * @brief       Driver for reading ISO 11784 & 11785 FDX-B Animal Tags
+ * 				Example: http://www.radtke.dk/mlx90109_schematic.pdf
```

Comment:
move down below next `@brief`, otherwise this describes the group but not the driver. Further:

- alignment/indention seems off
- could you `@see` for the URL link